home.html

```diff
@@ -0,0 +1,358 @@
+<!DOCTYPE html>
```

Comment:
This file should be named `index.html`, as that will be associated with the default path `/`